jib-core/src/main/java/com/google/cloud/tools/jib/image/json/HistoryEntry.java

```diff
@@ -32,7 +32,56 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class HistoryEntry implements JsonTemplate {
 
-  /** The RFC 3339 formatted timestamp at which the image was created. */
+  public static class Builder {
```

Comment:
nit: short javadoc

Reply:
I didn't see any other Builders in the code base with a javadoc (and I don't think it's necessary, this is pretty self-explanatory).